apps/rotm/views/content/confirmation.md

```diff
@@ -0,0 +1,9 @@
+##What happens next
+
+We have sent your anonymous report to the police team dealing with online terrorist material.
+
+[What did you think of this service? (takes 30 seconds)](https://eforms.homeoffice.gov.uk/outreach/Feedback.ofml?FormName=terrorism_reporting/)
+
+[Start a new report](/source)
+
+Attach one or more files to this issue
```

Comment:
WUT?!

Reply:
![](https://media.giphy.com/media/rsupaDCIpkaPe/giphy.gif)

I'll delete that